client/src/components/Screens/Login/Login.jsx

```diff
@@ -23,16 +23,22 @@ const Login = () => {
         email,
         password,
       });
+      console.log(response);
 
-      const { userToken } = response.data; // Extract user token from response
+      const { userToken, userId } = response.data; // Extract user token and userId from response
       console.log("Token received:", userToken);
-
       localStorage.setItem("userToken", userToken); // Store token in local storage
+      localStorage.setItem("userId", userId); // Store userId in local storage
+
       console.log(
         "Token stored in localStorage:",
         localStorage.getItem("userToken")
       );
+
       navigate("/home"); // Navigate to home page after successful login
+
+      // Redirect to Spotify login after successful login
+      window.location.href = `http://localhost:3000/auth/login?userId=${userId}`;
```

Comment:
Hardcode the local host is not the best practice.  It will cause an issue if another machine is calling it.  

Can we change it to `${window.location.protocol}//${window.location.hostname}:3000/auth/login?userId=${userId}`